### PR TITLE
Login & session

### PR DESCRIPTION
**Pull request by elhamamini**

### CODE REVIEW

File: app/components/login.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
```
Comment (ShelOrock) [inline app/components/login.js]: I'm working on UI

Comment (elhamamini) [reply app/components/login.js]: oh,sorry you are right, but i couldnt test it with out login component

Comment (ShelOrock) [reply app/components/login.js]: ok. we can figure out how to integrate my code into this tomorrow. For now I'll leave the PR open.

File: app/redux/index.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 import { combineReducers } from 'redux';
-
-const appReduer = combineReducers({});
+import authenticationReducer from './authentication';
+import usersReducer from './users';
+const appReduer = combineReducers({
```
Comment (ajmorozoff) [inline app/redux/index.js]: This is still typo

**Pull request merged**